docs/integration_resources/software/nuke.md

```diff
@@ -14,59 +14,107 @@ nshift.show()
+| *Bool*       | -              | Knob                                             | Boolean_Knob                                                 |                                                                                                     |
+| *Color*      | -              | Knob                                             | AColor_Knob                                                  |                                                                                                     |
+| *Dict*       | -              | Knob                                             | EvalString_Knob                                              | Require custom implementation                                                                       |
+| *Dir*        | -              | Knob                                             | File_Knob                                                    |                                                                                                     |
+| *Enumerator* | -              | Knob                                             | Enumeration_Knob                                             |                                                                                                     |
+| *FileIn*     | -              | Knob                                             | File_Knob                                                    |                                                                                                     |
+| *FileOut*    | -              | Knob                                             | File_Knob                                                    |                                                                                                     |
+| *Float*      | -              | Knob                                             | Double_Knob                                                  |                                                                                                     |
+| *Instance*   | image*         | Input/Output<br/>Knob<br/>Knob                   | - <br/>Channel_Knob <br/>File_Knob                           | Provides the input image like a single image. (Current Frame)                                       |
+| *Instance*   | images*        | Output<br/>Knob<br/>Knob                         | - <br/>File_Knob                                             | Provides the input image like a single image. (Current Frame)                                       |
+| *Instance*   | imageSequence* | Input/Output<br/>Knob<br/>Knob<br/>Knob<br/>Knob | - <br/>Channel_Knob <br/>File_Knob<br/>Int_Knob<br/>Int_Knob | Provides the input image like a image sequence rendering the full frame range provided in the knob. |

```

Comment:
This is an implementation stress point. PIL instances are picklable, therefore we do not need these to be kInstance plugs. They can be kObject types as well.

Reply:
After discussing in a meeting this is the implementation allowed for the moment. We will revisit this when we decide what to do with the image plugs. resolving